doc/HOWTO_RELEASE.rst

```diff
@@ -13,13 +13,13 @@ Useful info can be found in the following locations:
 
 * **NumPy docs**
 
-  - https://github.com/numpy/numpy/blob/main/doc/HOWTO_RELEASE.rst
-  - https://github.com/numpy/numpy/blob/main/doc/RELEASE_WALKTHROUGH.rst
-  - https://github.com/numpy/numpy/blob/main/doc/BRANCH_WALKTHROUGH.rst
+  - `HOWTO_RELEASE.rst <https://github.com/numpy/numpy/blob/main/doc/HOWTO_RELEASE.rst>`_
+  - `RELEASE_WALKTHROUGH.rst <https://github.com/numpy/numpy/blob/main/doc/RELEASE_WALKTHROUGH.rst>`_
+  - `BRANCH_WALKTHROUGH.rst <https://github.com/numpy/numpy/blob/main/doc/BRANCH_WALKTHROUGH.rst>`_
 
 * **Release scripts**
 
-  - https://github.com/numpy/numpy-vendor
+  - `numpy-vendor <https://github.com/numpy/numpy-vendor>`_
```

Comment:
We could probably just remove this section, that repo hasn't been touched in 10 years and is quite out of date for windows builds.

@rgommers do we have a policy on removing/deprecating outdated repos? I can see keeping some of them around for historical reasons, but it would be nice if we could put them in storage.

Reply:
We'll keep such repos around forever, but they're archived. That is already the case for this one:

![image](https://github.com/user-attachments/assets/4561e633-0ccb-424c-bbcd-f7a778dfc76d)

So simply removing the link here will be fine.

Reply:
@charris @rgommers Thanks for reviewing! I have removed the numpy-vendor link.